src/xsds/gfcpp-cache-9.0.xsd

```diff
@@ -33,7 +33,7 @@ limitations under the License.
       version="9.0">
 
 The contents of a declarative XML file correspond to APIs found in the
-com.gemstone.gemfire.cache and com.gemstone.gemfire.cache.client
+org.apache.geode.cache and org.apache.geode.cache.client
```

Comment:
Have you verified that these are the correct packages after GemFire became Geode?